DESCRIPTION

```diff
@@ -1,12 +1,48 @@
 Package: indexwc
-Title: What the Package Does (One Line, Title Case)
-Version: 0.0.0.9000
-Authors@R: 
-    person("First", "Last", , "first.last@example.com", role = c("aut", "cre"),
-           comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: What the package does (one paragraph).
-License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
-    license
+Title: Run indices for West Coast Groundfish stock assessments
+Version: 0.1
+Authors@R: c(
+    person(given = c("Kelli", "F."),
+      family = "Johnson",
+      role = c("aut", "cre"),
+      email = "kelli.johnson@noaa.gov",
+      comment = c(ORCID = "0000-0002-5149-451X")),
+    person(given = c("Chantel", "R."),
+      family = "Wetzel",
+      role = "aut",
+      email = "chantel.wetzel@noaa.gov",
+      comment = c(ORCID = "0000-0002-7573-8240"))
+    )
+Maintainer: Kelli F. Johnson <kelli.johnson@noaa.gov.
+Description: Runs indices of abundance for survey data from the NWFSC
+	and fishery-dependent catch-per-unit-effort.
+License: MIT + file LICENSE
 Encoding: UTF-8
+LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.2
+RoxygenNote: 7.2.3
+Depends:
+  R (>= 4.0)
+Imports:
+    sdmTMB,
```

Comment:
You may want to tie this to at least some minimum version number of sdmTMB, likely based on a CRAN release. https://cran.r-project.org/package=sdmTMB

There's one important bug fix since that release: sigma_E in the 2nd part of a delta model was getting reported as the sigma_E of the 1st part in `print()`. It was a printing problem (not fitting problem) that crept in when time-varying sigma_E was added.

The latest GitHub sdmTMB now prints helpful information on the anisotropic range thanks to work by @jdunic.

We'll update the CRAN version sometime soon.

Reply:
We now depend on the latest version in main